### DISCUSSION TITLE
v3.0.1 With fix for pillow v10 has been released

### DISCUSSION THREAD

**Discussion by coolapso** [streamdeck-linux-gui/streamdeck-linux-gui#18] [Announcements]
Verson 3.0.1 has now been released and published to pypi. 

This version contains the fix for the long awaited fix for python pillow v10. 

You can now update python-pillow :D 

All release details here: https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/releases/tag/v3.0.1

AUR Packages hopefully soon to follow.

**Comment by coolapso** [comment]
A very special thanks to @irthomasthomas for bringing his patch here.

**Comment by xurei** [comment]
I noticed that the CHANGELOG does not reference this version. 
Are we going to stop using it ?

**Reply by coolapso** [reply]
changelog is no longer in use ... changes are instead visible in the releases page: https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/releases
